OrbitBase/include/OrbitBase/Platform/Linux/Profiling.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_PLATFORM_LINUX_PROFILING_H_
+#define ORBIT_BASE_PLATFORM_LINUX_PROFILING_H_
+
+#include <stdint.h>
+#include <sys/syscall.h>
+#include <time.h>
+#include <unistd.h>
+
+#include <string>
+
+[[nodiscard]] inline uint64_t MonotonicTimestampNs() {
+  timespec ts;
+  clock_gettime(CLOCK_MONOTONIC, &ts);
+  return 1'000'000'000llu * ts.tv_sec + ts.tv_nsec;
```

Comment:
nit: LL (harder to confuse them with 1)

Reply:
Ah sure. I favored the implementation from LinuxTracingUtils, will make the change.